protostar/starknet/cheaters/contracts.py

```diff
@@ -288,3 +301,45 @@ async def call(
             general_config=StarknetGeneralConfig(),
         )
         return result.retdata
+
+    async def mock_call(
+        self,
+        contract_address: Address,
+        function_name: str,
+        ret_data: CairoOrPythonData,
+    ) -> Callable:
+        ret_data_cairo = await self._transform_calldata_to_cairo_data_by_addr(
```

Comment:
I dont like this transformation being part of the cheaters logic. I would add `get_contract_class_by_addr` to cheatable state and moved remaining logic to  `protostar.starknet.data_transformer`, what do you think?